README.md

```diff
@@ -54,10 +57,24 @@ As per [µKanren: A Minimal Functional Core for Relational Programming](http://w
 
 ### Example
 ```go
+package main
+
+import (
```

Comment:
I thought to only add a full runnable example at first, then leave out the boilerplate further down. If you think this should also have it, again for consistency shouldn't all of the examples have it?

Reply:
I was adding the additional code to show it was separate package, not to duplicate all of the code. Not sure it has to be completely consistent, more that it needs to demonstrate any important ideas.

Reply:
I think I left enough in to show the point, removed the full program from the last part of readme since the actual examples are in the repo now.